Makefile

```diff
@@ -148,6 +148,14 @@ $(warning No Mixpanel token found (ANALYTICS_MIXPANEL_TOKEN is not set))
 endif
 
 # ---------------------------------------------------------------------
+# Updates
+# ---------------------------------------------------------------------
+
+ifdef DISABLE_UPDATES
+$(warning Update notification dialog was disabled (DISABLE_UPDATES is set))
```

Comment:
Minor grammar nit: strange mix here of `was` and `is` - perhaps `was` could be changed to `has been` ?